src/domain/ecoverse/ecoverse.resolver.queries.ts

```diff
@@ -63,10 +63,7 @@ export class EcoverseResolverQueries {
     return this.ecoverseService.getContext();
   }
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.